neusomatic/python/generate_dataset.py

```diff
@@ -970,7 +975,7 @@ def find_records((work, split_region_file, truth_vcf_file, pred_vcf_file, ref_fi
                         records.append(rr + [str(i)])
                         i += 1
         records_bed = pybedtools.BedTool(map(lambda x: pybedtools.Interval(
-            x[0], x[1], x[1] + len(x[2]), x[2], x[3], x[4]), records))
+            x[0], x[1], x[1] + len(x[2]), x[2], x[3], x[4]), records)).saveas()
```

Comment:
Where would this be saved and with what name? Why is this `saveas` needed?

Reply:
@marghoob Sometimes pybedtools behave like a iterator (usually when you are using pybedtools.Interval as in here) so to avoid losing it, we can save it. saveas() with no argument will save it in pybedtools tmp_dir. Then we can still access it later.